pages/common/minetest.md

```diff
@@ -8,9 +8,9 @@
 
 `minetest`
 
-- Start minetest in server mode:
+- Start minetest in server mode by hosting the specified world:
 
-`minetest --server`
+`minetest --server --world {{name}}`
```

Comment:
this works only with `--server` right? why not add an extra example for a specific world?

Reply:
Yes and `--server` doesn't work at all without specifying a world.